components/TechyApi.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+import { View, Text, TouchableOpacity } from 'react-native';
```

Comment:
`View` and `TouchableOpacity` are not used, so it's better to remove them here (you don't have to import them when they are styled with styled-components).